src/controller/app_routes.rs

```diff
@@ -253,11 +366,23 @@ mod tests {
                 Routes::new().add("multiple3", patch(action)),
             ]);
 
-        for route in app_router.collect() {
-            assert_debug_snapshot!(
-                format!("[{}]", route.uri.replace('/', "[slash]")),
-                format!("{:?} {}", route.actions, route.uri)
-            );
+        let routes = app_router.collect();
+        let expected_routes = vec![
+            ("/", vec![Method::GET]),
+            ("/normalizer/no-slash", vec![Method::GET]),
+            ("/normalizer", vec![Method::POST]),
+            ("/normalizer/loco/rs", vec![Method::DELETE]),
+            ("/normalizer/multiple-start", vec![Method::HEAD]),
+            ("/normalizer/multiple-end", vec![Method::TRACE]),
+            ("/multiple1", vec![Method::PUT]),
+            ("/multiple2", vec![Method::OPTIONS]),
+            ("/multiple3", vec![Method::PATCH]),
+        ];
+
+        assert_eq!(routes.len(), expected_routes.len());
+        for (i, route) in routes.iter().enumerate() {
+            assert_eq!(route.uri, expected_routes[i.clone()].0);
+            assert_eq!(route.actions, expected_routes[i].1);
```

Comment:
The test is a bit hard to follow and maintain. It loops over one vector while expecting values from another, making it less intuitive.

Additionally, if the order changes, updating the test becomes cumbersome. I prefer using a snapshot here since it avoids the need to worry about ordering.

Reply:
Okay I'll revert this change.